static/scripts/topicEdit.js

```diff
@@ -1065,7 +1065,7 @@ class TopicH5P extends TopicBlock {
 		const { parentType, parentId } = this.props;
 
 		const editorPopup = window.open(
-			`/h5p/editor/${id ?? ''}?parentType=${parentType}&parentId=${parentId}&inline=1`,
+			`/h5p/editor/${id ?? 'new'}?parentType=${parentType}&parentId=${parentId}&inline=1`,
```

Comment:
where does the "new" path comes from?
also, where is this URL mapped? Should it not have some sort of configurable domain? Is not a page in legacy client, or?